### PR TITLE
Document write caching for Kubernetes

### PR DESCRIPTION
**Pull request by JakeSCahill**
Preview: https://deploy-preview-430--redpanda-docs-preview.netlify.app/24.1/manage/kubernetes/k-manage-topics/#configure-write-caching

### CODE REVIEW

File: modules/manage/pages/kubernetes/k-manage-topics.adoc [hunk 1]
```diff
@@ -123,6 +123,108 @@ The cleanup policy determines how to manage the partition log files when they re
...
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --values write-caching.yaml --reuse-values
+```
+
+--set::
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set config.cluster.write_caching_default=true
```
Comment (RafalKorepta) [inline modules/manage/pages/kubernetes/k-manage-topics.adoc:190]: I wonder why 
```
helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
  --values write-caching.yaml --reuse-values
```
has `--reuse-values`

But this command misses it. Maybe It's nothing to worry about, but I would need to check.

**Review by RafalKorepta — APPROVED** [review]

**Comment by micheleRP** [comment]
@JakeSCahill please check the updates Nicolae made last week in PR #425

**Review by Feediver1 — APPROVED** [review]
LGTM

**Comment by Feediver1** [comment]
@JakeSCahill Please look at the comments from @nicolaferraro 's PR--looks like some wording differences wrt to fsync.

File: modules/manage/pages/kubernetes/k-manage-topics.adoc [hunk 2]
```diff
@@ -123,6 +123,108 @@ The cleanup policy determines how to manage the partition log files when they re
 
 If the cleanup policy is `delete` or `compact,delete` you can <<Delete records from a topic>>.
 
+=== Configure write caching
+
+Write caching is a relaxed mode of xref:develop:produce-data/configure-producers.adoc#acksall[`acks=all`] that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write.
```
Comment (JakeSCahill) [inline modules/manage/pages/kubernetes/k-manage-topics.adoc]:
```suggestion
Write caching is a relaxed mode of xref:develop:produce-data/configure-producers.adoc#acksall[`acks=all`] that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to be written to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write.
```

**Pull request merged**